### DISCUSSION TITLE
Regarding Industry Sessions status in Future Ready Talent

### DISCUSSION THREAD

**Discussion by crescent118** [github/india#424] [Students]
Hello there,
I am Rishika Sinha here, a student who had registered for internship in Future Ready Talent co-partnered by Microsoft and GitHub. I have successfully completed my Orientation program but when I am watching the videos under Industry Session tab, the status is showing incomplete though I have watched the videos at one go only and as directed there. And without the completion tag, I won't be able to progress and move to the next step.

Please look into this matter.

Thank You

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by TheSuranaverse** [comment]
@crescent118
Dear Rishika,

Please join the community above & ping us there. We will help you out with everything in this Internship. For any doubt/query do let me know.

Thank You!

**Reply by ssritgithub** [reply]
same issue sir
